rocksdb_install.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -e
+
+if [[ "$OSTYPE" == "linux-gnu"* ]] && [[-x "$(command -v apt-get)" ]]]; then 
+        sudo apt-get install libgflags-dev libsnappy-dev zlib1g-dev libbz2-dev liblz4-dev libzstd-dev
+elif [[ "$OSTYPE" == "darwin"* ]]; then
+        brew deps --tree --1 rocksdb
```

Comment:
Not exactly what I meant, run this on your system to get the dependencies, then add a
brew install
line here with all the dependencies that rocksdb needs

We are building rocksdb ourselves in this project, we just need to make sure all its dependencies are installed externally on the system